snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -144,7 +144,7 @@ init_config:
   #   isilon:
   #     definition_file: isilon.yaml
   #   apc-ups:
```

Comment:
```suggestion
  #   apc_ups:
```

Reply:
I think we need also to fix the profile name to be consistent with the default profile name that is based on the file name: https://github.com/DataDog/integrations-core/blob/86e9dc09f5a1829a8e8bf1b404c4fb73a392e0e5/snmp/datadog_checks/snmp/utils.py#L123-L126